kafka/source/pkg/reconciler/kafkasource.go

```diff
@@ -229,6 +233,145 @@ func checkResourcesStatus(src *v1alpha1.KafkaSource) error {
 	return nil
 }
 
+// //Uses KEDA API
```

Comment:
Do we want all this commented code committed?

Reply:
(ah nvm, I see it's still marked a WIP)

Reply:
Removed it for now